itchiodl/game.py

```diff
@@ -64,7 +64,10 @@ def load_downloads(self, token):
                 headers={"Authorization": token},
             )
         j = r.json()
-        for d in j["uploads"]:
+        # This converts a null j.uploads into an empty dictionary and uses the safer .get accessor
+        if j.get("uploads") is None:
```

Comment:
could this be replaced with `"uploads" in j`

Reply:
I don't believe that this:

`for "uploads" in j: 
`
is valid python, since you're putting a literal where it expects to assign the iterator

Reply:
I just looked at this again and realized what you meant, fixed now.